Gemfile

```diff
@@ -24,7 +24,7 @@ end
 gem 'dhasher'
 gem 'discard', :git => 'https://github.com/jhawthorn/discard', :branch => 'master'
 gem 'jbuilder', '~> 2.0'
-gem 'manageiq-api-common', '~> 1.1'
+gem 'manageiq-api-common', :git => 'https://github.com/ManageIQ/manageiq-api-common', :branch => 'master'
```

Comment:
@syncrou This change was done so that we don't have to wait for the common gem to arrive in ruby gems.org.
The 2.0 change that was pushed to rubygems.org doesn't have this PR https://github.com/ManageIQ/manageiq-api-common/pull/126 yet. Once its there we can change this to 2.0